README.md

```diff
@@ -280,6 +280,72 @@ SLF4J Logger.
   metricRegistry.addReporter(reporter);
 ```
 
+#### Histograms
+
+In the current implementation, clients must define the distribution buckets and associate them in the reporter with the name of the metric to be histogrammed.
+
+There two types of distribution buckets available:
+- `DistributionBucket` represented by a primitive `long` array.
+- `DoubleValuedDistributionBucket` represented by a primitive `double` array
+  
+##### DistributionBucket
+Used to represent the distribution of a integer value. For example time spent in milliseconds or size of a messaging queue.
+
+For a given distribution array: [0, 10, 100, 500, 1000], the buckets would be like:
+* [0,10) for values 0-9
```

Comment:
Since we measure time in nanos, would it be more appropriate to have an example where the buckets are sized for tracking ranges in micros / millis as very few things complete in under 100 nanos that one would measure?

Reply:
Ohh... you mean the code example.  Agreed. Fixed it to measure latencies in millis. Please, take a look.

Reply:
No, I was referring to bucket definitions. We should encourage measuring times in nanos for consistency.

Reply:
Okay. I have fixed that. Please take a look.